content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
-data from the Petstore API to. Let us choose the *E-Mail* component.
+Next, we need to retrieve a data sample and continue:
 
-![Choosing the E-mail component](/assets/img/getting-started/first-flow/getting-started-flow-010.png "Choosing the E-mail component")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_13.png)
 
-We can find it in the list of components. The E-Mail component has no credentials,
-so we jump directly to the mapping of data between Petstore and E-Mail components.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_14.png)
 
-![Input fields for E-mail component](/assets/img/getting-started/first-flow/getting-started-flow-011.png "Input fields for E-mail component")
+Let's finalize the step by adding a description and clicking "Finish step":
 
-The screenshot above demonstrates the mapping of data between the Petstore trigger
-and E-Mail action. The E-Mail components requires a configuration of 3 following
-fields to work properly:
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_15.png)
 
-*   **To** field: expects an e-mail address wrapped by colons, such as `"your@email.com"`. E-Mail component will use this field to send the data to. Note that if you enter the address without wrapping them by `" "` it will get rejected.
+Now we will add another action step:
 
-*   **Subject** field: expects email's subject as a `string` expression. Here we use some advance JSONata expression: `$count(pets.id) & " " & "new pets found"`
-Please note that we will not dive into JSONata at the moment, copy and paste this expression into the *Subject* field.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_16.png)
 
-*   **Body** field: expects email's body as a `string`. Here is another JSONata expression to copy and paste:
-``
-"Darling, I found some adorable pets for us. Here are their names: " & $join(pets.name, ", ") &". Which one shall I buy?"
-``
+Optionally, set a name for the step. We're going to use Email Component for our third step. Select it and click "Choose Email":
 
-Click *Continue* after filling-in the form to continue.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_17.png)
 
-![Retrieve a sample from Email](/assets/img/getting-started/first-flow/getting-started-flow-012.png "Retrieve a sample from Email")
+Email Component sends an email when triggered, so you should just fill in the details as you would for an email. Also, you can cancel error message on the event of sending fail. After you've entered all the data, click "Continue":
 
-On the next page click on *Retrieve sample from E-Mail* to retrieve the sample.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_18.png)
 
-![Continue with the Email components' sample](/assets/img/getting-started/first-flow/getting-started-flow-013.png "Continue with the Email components' sample")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_19.png)
```

Comment:
Maybe a few words about the mapping results? Why do I see `RRRAAAWWW`? I guess we need to explain samples above a little bit or mention here that the user will understand it by reading another article linked.